server/passport.ts

```diff
@@ -251,6 +253,11 @@ function setupPassport(models: DB) {
     callbackURL: GITHUB_OAUTH_CALLBACK,
     passReqToCallback: true,
   }, async (req: Request, accessToken, refreshToken, profile, cb) => {
+    const str = '&state='
```

Comment:
What's this session validation do for Github?

Reply:
It's simply a security feature. You pass it in when you start the authentication process and GitHub will add the state param to the callback URL.